plugins/inputs/nats_consumer/nats_consumer.go

```diff
@@ -4,12 +4,11 @@ package nats_consumer
 import (
 	"context"
 	_ "embed"
-	"fmt"
+	"github.com/influxdata/telegraf/plugins/common/parsing"
+	"github.com/nats-io/nats.go"
```

Comment:
Please group the import as _built-in_ libraries, _3rd party_ libraries and _telegraf internal_ imports. Each group separated by a newline with alphabetical order within each group.